### DISCUSSION TITLE
[A-L] (8 Febbraio 2022) Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#359] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/153036719-d2d350d0-b80c-47ac-a080-5626ce77464e.png)

**Comment by mafelix02** [comment]
![1](https://user-images.githubusercontent.com/91395167/153733745-cd6e8b6f-4f21-45da-8207-2a5070514748.PNG)
![2](https://user-images.githubusercontent.com/91395167/153733748-a5018213-cd46-4ead-a5cd-1b2c3fe62bbc.PNG)

**Comment by matypist** [comment]
Mi ritrovo con la soluzione di @mafelix02 e non avrei saputo rappresentarla meglio di così :+1:

**Comment by francesco-alberghina** [comment]
Soluzione analoga a @mafelix02 ma con le 3 proposizioni messe in ordine differente

![immagine](https://github.com/sapienzastudentsnetwork/mmi2122/assets/68157574/dcadc7cd-496b-47cb-9762-10b95d92eb56)
